js/components/adPanel.js

```diff
@@ -72,7 +72,16 @@ var AdPanel = React.createClass({
       adPlaybackInfo = adPlaybackInfo + ": (" + currentAdIndex + "/" + totalNumberOfAds + ")";
     }
 
-    var remainingTime = Utils.formatSeconds(Math.max(0, parseInt(this.props.adVideoDuration - this.props.currentPlayhead)));
+    var isLive = this.props.currentAdsInfo.currentAdItem.isLive;
+
+    var remainingTime;
+    if (isLive) {
+      remainingTime = parseInt((this.props.adStartTime + this.props.adVideoDuration * 1000 - new Date().getTime())/1000);
```

Comment:
can you surround these remainingTime calculations with a check for this.props?  Also, what if the number don't exist and we get a NaN. Can you do a check at the end and set it to 0 if it's NaN